master/buildbot/status/web/build.py

```diff
@@ -31,6 +31,20 @@
 from buildbot.status.web.tests import TestsResource
 from buildbot import util, interfaces
 from buildbot.status.results import RESUME, EXCEPTION
+from buildbot.steps.trigger import Trigger
+
+
+@defer.inlineCallbacks
+def prepare_trigger_links(current_step):
```

Comment:
The same, we need tests:)

Reply:
I have written `test_prepare_trigger_links_for_nontrigger` and `test_prepare_trigger_link_for_trigger` I had to use a lot of code for running fakedb in `test_status_buildstep.py`